test/e2e/framework.go

```diff
@@ -1613,12 +1617,23 @@ func (data *TestData) GetGatewayInterfaceName(antreaNamespace string) (string, e
 	return antreaDefaultGW, nil
 }
 
-func (data *TestData) mutateAntreaConfigMap(mutatingFunc func(data map[string]string), restartController, restartAgent bool) error {
+func (data *TestData) mutateAntreaConfigMap(controllerPair []configPair, agentPair []configPair, restartController, restartAgent bool) error {
```

Comment:
maybe `controllerChanges` and `agentChanges` would be better names than `controllerPair` and `agentPair`, and I feel like `configChange` is a better name than `configPair` for the type

Reply:
Updated.